index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-bar">
+        <a href="#" class="logo">My logo</a>
+        <nav class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></nav> 
+        </header>
+        <div class="image-icons">
+            <ul>
+            <li><div class="left-ellipse"><img src="images/Ellipse11.png" alt=""></div></li>
+            <li><div class="center-ellipse1"><img src="images/Ellipse5.png" alt=""></div></li>
+            <li><div class="center-rectangle"><img class="center-rect" src="images/Rectangle7.png" alt=""></div></li>
+            <li><div class="center-ellipse2"><img src="images/Ellipse08.png" alt=""></div></li>
+            <li><div class="right-ellipse"><img src="images/Ellipse3.png" alt=""></div></li>
+            <li><div class="right-rectangle"><img src="images/Rectangle4.png" alt=""></div></li>
+        </ul>
+        </div>
```

Comment:
- [ ] For background image you try to import images from images folder but unfortunately there is no images folder in your Repo, so please try to fix it.